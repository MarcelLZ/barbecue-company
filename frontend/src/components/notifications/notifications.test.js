import React from 'react'
import { mount } from 'enzyme'

import Notifications from '.'

describe('<Notifications />', () => {
  it('should render correctly', () => {
    const wrapper = mount(<Notifications />)

    expect(wrapper).toMatchSnapshot()
  })
})
