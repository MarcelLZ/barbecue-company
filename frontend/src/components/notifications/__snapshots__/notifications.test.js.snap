// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Notifications /> should render correctly 1`] = `
<Notifications>
  <div>
    <_class
      options={Object {}}
    >
      <div
        id="notification-wrapper"
      />
    </_class>
  </div>
</Notifications>
`;
